html-aam/index.html

```diff
@@ -1257,6 +1257,328 @@ <h4 id="el-button">`button`</h4>
             </tr>
           </tbody>
         </table>
+        <h4 id="el-button-command-togglepopover">`button` <span class="el-context">(`command` attribute in the Toggle popover state)</span></h4>
```

Comment:
Is there a difference in what we expose for toggle vs show? Can we the combine mapping entries for those into one?